scout/adapter/mongo/variant.py

```diff
@@ -779,7 +779,6 @@ def evaluated_variants(self, case_id, institute_id):
         query = {
             "$and": [
                 {"variant_id": {"$in": variant_ids}},
-                {"institute": institute_id},
```

Comment:
variant_ids are collected from the events already, so I guess no need having institute again here.
Additionally this institute parameter here hinder finding variants with classification when the case belongs to another institute - this was the original problem 👍🏻 

The case belongs to cust000, but has cust083 as a collaborator
<img width="361" alt="image" src="https://github.com/user-attachments/assets/270a2b18-a620-4d8c-bfc2-3ca50de87713">

### Evaluated vars are not returned when this query is runned with cust000:

<img width="1048" alt="image" src="https://github.com/user-attachments/assets/2aafcee8-4147-4a2b-a3b3-ca5b3fd9a6d7">


### But they are found when searching for cust083:

<img width="1005" alt="image" src="https://github.com/user-attachments/assets/3a361d2d-7a41-4487-92ca-7729d2e51851">